PyInstaller/utils/hooks/conda.py

```diff
@@ -193,3 +194,67 @@ def collect_dynamic_libs(name, dest=".",
                 files.append((str(CONDA_ROOT / file), dest))
 
     return files
+
+
+PYTHONPATH_PREFIXES = []
+for path in sys.path:
+    path = Path(path)
+    try:
+        PYTHONPATH_PREFIXES.append(path.relative_to(sys.prefix))
+    except ValueError:
+        pass
+
+
+def _get_package_name(file):
+    """Determine the package name of a Python file in ``sys.path``.
+
+    :param file: A Python filename relative to Conda root (sys.prefix).
+    :return: Package name or None.
+
+    This function only considers single file packages e.g. ``foo.py`` or
+    top level ``foo/__init__.py``\\ s. Anything else is ignored (returning
+    ``None``).
+    """
+    file = Path(file)
+    if file.name == "__init__.py":
```

Comment:
This fails if the `__init__` module is an extension module. See 0b496be42c67591576c1943635083ee6bc2940da
```suggestion
    if file.stem == "__init__" and file.suffix in compat.ALL_SUFFIXES:
```